lib/unleash/constraint.rb

```diff
@@ -38,8 +42,6 @@ def initialize(context_name, operator, value = [], inverted: false, case_insensi
     end
 
     def matches_context?(context)
-      Unleash.logger.debug "Unleash::Constraint matches_context? value: #{self.value} context.get_by_name(#{self.context_name})" \
-        " #{context.get_by_name(self.context_name)} "
```

Comment:
When debugging, it is nice to have this message come up. 

So I would avoid dropping it, if i could...

Reply:
It is not dropped, its moved. This message is what is raising the KeyError, so if it is up here we never get to graceful handling of this.

Reply:
why not keep it in both places? 

This is debug logging to help us get an idea of the state the SDK is in, if we ever need to debug it when it is deployed out in the wild.

It can be handy to have a confirmation in the log that we have made it so far in the code.